### ISSUE TITLE
Splitting traffic under routes is a bit difficult to understand

### ISSUE THREAD

**Issue by benjaminapetersen** [openshift/origin-web-console#1070]
Pulled from original comment on [this PR](https://github.com/openshift/origin-web-console/pull/1068#issuecomment-268878285).
A few observations:

- Checking "split traffic" is a bit jumpy.  Couple things I struggled with here:
- The slider works well, but once I click "edit weights as integers" I couldn't go back to the slider without refreshing the page.
- When I switch to "edit weights as integers" it took me a while to figure out why I had a dropdown with only one service, even though the slider shows 2 services:
![screen shot 2016-12-22 at 2 47 03 pm](https://cloud.githubusercontent.com/assets/280512/21438108/7f0d3aa6-c855-11e6-9042-57bd603fa47d.png)
    I think its because the first service shows up above and the "weight" input now appears next to it.  I had to think about that a bit as I expected everything I was interacting with to appear below the heading.  I wonder if we could leave the above selector to be exclusive to "service to route to" and just list them all again under "split traffic" with the weight inputs (somehow show the primary service linked to the above "service to route to").
- Once I added additional services beyond 2, it was a little easier to understand.  

@spadgett @jwforres

**Comment by spadgett** [comment]
The interaction is probably less obvious now that we added the slider. Before, you picked a service and entered a weight in the input beside it as one step. Now if the page switches from the slider to integer weights it's not obvious what changed since the inputs are above where you clicked and might even be off screen.

**Comment by benjaminapetersen** [comment]
Yup, I think thats what confused me, now that it jumps a bit its hard to follow which inputs are actually new.
